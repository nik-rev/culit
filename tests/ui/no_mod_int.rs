use culit::culit;

mod custom_literal {}

#[culit]
fn foo() {
    let a = 10km;
}
