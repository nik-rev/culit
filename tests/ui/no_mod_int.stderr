error[E0433]: failed to resolve: could not find `integer` in `custom_literal`
 --> tests/ui/no_mod_int.rs:8:1
  |
8 | #[culit]
  | ^^^^^^^^ could not find `integer` in `custom_literal`
  |
  = note: this error originates in the attribute macro `culit` (in Nightly builds, run with -Z macro-backtrace for more info)
